### ISSUE TITLE
Split demo and project

### ISSUE THREAD

**Issue by ai** [emilbonnek/lit-nanostores#2]
Right now it is hard to separate `lit-nanostores` code and example app code

**Comment by emilbonnek** [comment]
Yup, I am aware :) I have another repo where I have just the reactive controller and the decorators. I am still playing a bit around with it to see what works best.

**Comment by emilbonnek** [comment]
It is at https://github.com/emilbonnek/nanostores-lit
